File: Completo/ws16StoreApp/StoreApp_py_js_java/StoreApp.js/StoreApp/ec.edu.espe.store.view/Store.js
import { Product } from '../ec.edu.espe.store.model/Product.js';
import { Tax } from '../utils/Tax.js';

function main() {
    let id;
    let description;
    let price;
    let pvp;
    let product;

  
    id = 1;
    description = "Computer";
    price = 100;

    pvp = Tax.computeTotal(price, 15);
    product = new Product(id, description, price);
    console.log("product---> " + product.toString());

   
    let product2;
    id = 2;
    description = "Mouse";
    price = 1000;

    pvp = Tax.computeTotal(price, 15);
    product2 = new Product(id, description, price);
    console.log("product---> " + product2.toString());
}

main();

File: Completo/ws16StoreApp/StoreApp_py_js_java/StoreApp.Java/StoreApp/StoreApp/src/ec/edu/espe/store/view/Store.java

package ec.edu.espe.store.view;

import ec.edu.espe.store.model.Product;
import utils.Tax;

/**
 *
 * @author LABS-ESPE
 */
public class Store {
    public static void main(String[] args) {
        int id;
        String description;
        float price;
        float pvp;
        Product product;
        
        id=1;
        description="Computer";
        price=100;
        
        pvp= Tax.computeTotal(price,15F);
        
        product= new Product(id, description, price);
        
        System.out.println("product--->"+product);
        
        Product product2;
        id=2;
        description="Mouse";
        price=1000;
        
        pvp= Tax.computeTotal(price,15F);
        
        product= new Product(id, description, price);
        
        System.out.println("product--->"+product);
        
    }
    
}
